cmake/modules/Buildpmem.cmake

```diff
@@ -12,8 +12,8 @@ function(build_pmem)
   set(PMDK_LIB "${PMDK_SRC}/${PMDK_LIB_DIR}")
 
   ExternalProject_Add(nvml_ext
-      GIT_REPOSITORY "https://github.com/ceph/nvml.git"
-      GIT_TAG "dd622819dd4ee97d3920f913c70be"
+      GIT_REPOSITORY "https://github.com/pmem/pmdk.git"
```

Comment:
s/pmem/ceph/ see https://github.com/ceph/pmdk

Reply:
and please note the rationale in the commit message, for instance

1. nvml is renamed to pmdk, and
2. we need to bump to the latest stable release: 1.7 ..

Reply:
> s/pmem/ceph/ see https://github.com/ceph/pmdk

@tchaikov Hi Kefu, is this what you want to update PMDK repository in Ceph? https://github.com/ceph/pmdk/pull/2

Reply:
as discussed offline. i just sync'ed ceph/pmdk. so we just need to point `GIT_REPOSITORY` to it and update `GIT_TAG` to `1.8`.

Reply:
+1 to use named tag `stable-1.8` if possible instead of commit hash

Reply:
@CongMinYin could you take this opportunity to s/nvml_ext/pmdk_ext/ as well?

Reply:
> @CongMinYin could you take this opportunity to s/nvml_ext/pmdk_ext/ as well?

@tchaikov Yes, I am testing it now.